src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -62,11 +62,11 @@ public static InstanceMappingFileSettings InstanceMappingFile(this RoutingSettin
         }
 
         /// <summary>
-        /// Moves expired messages using the "time to reach queue" setting to the dead letter queue instead of discarding them.
+        /// Moves messages that have expired their TimeToBeReceived to the dead letter queue instead of discarding them.
```

Comment:
How about "Moves messages that have **exceeded** their TimeToBeReceived" instead? Using "expired" in this context sounds awkward.